webrtc/RTCPeerConnection-iceConnectionState.https.html

```diff
@@ -371,4 +371,29 @@
   assert_array_equals(pc2.iceStates, ['new', 'checking', 'connected']);
 }, 'Responder ICE connection state behaves as expected');
 
+/*
+  Test case for step 11 of PeerConnection.close().
+  ...
+  11. Set connection's ICE connection state to "closed". This does not invoke
+      the "update the ICE connection state" procedure, and does not fire any
+      event.
+  ...
+*/
+promise_test(async t => {
+  const pc1 = new RTCPeerConnection();
+  t.add_cleanup(() => pc1.close());
+  const pc2 = new RTCPeerConnection();
+  const stream = await getNoiseStream({ audio: true });
+  t.add_cleanup(() => stream.getTracks().forEach(track => track.stop()));
+
+  stream.getTracks().forEach(track => pc1.addTrack(track, stream));
+  coupleIceCandidates(pc1, pc2);
```

Comment:
We should use `exchangeIceCandidates` here. There's no spec support for requiring ice candidates be queued to avoid intermittents. That's a Chrome bug that should not dictate wpt tests. But I see this is prevalent, so I've opened https://github.com/web-platform-tests/wpt/issues/19866  for that.

Reply:
We should use coupleIceCandidates until we have a test that solidly exercises the case that causes the issue.

Reply:
Agree. The fiddle in https://github.com/web-platform-tests/wpt/issues/19866#issuecomment-546114464 should make a good test.

Reply:
Thanks for your review. `coupleIceCandidates` has been changed to `exchangeIceCandidates`.

Reply:
Updated this PR to address the comment above. Could you please take a look and help merge it? Thanks.

Reply:
Lgtm